emoji_message.py

```diff
@@ -4,17 +4,17 @@
 class EmojiMessage:
     """Constructs emoji report message."""
 
-    def __init__(self, channel, icon_emoji, event_type):
+    def __init__(self, report_channel, icon_emoji, event_type):
         """Initialize EmojiMessage.
 
         Created with the channel to send the message to, the event type that triggered the callback and the emoji to
         report on.
 
-        :param channel: The channel to send the emoji message to
+        :param report_channel: The channel to send the emoji message to
         :param icon_emoji: The name of the emoji
         :param event_type: The event that has happened, 'add' or 'remove'
         """
-        self.channel = channel
+        self.report_channel = report_channel
         self.username = 'automod'
```

Comment:
We should probably be moving stuff like this to a constant file somewhere....eventually.